docs/src/main/asciidoc/transaction.adoc

```diff
@@ -367,10 +367,13 @@ which is a CDI bean and can be ``@Inject``ed.
 
 In cloud environments where persistent storage is not available, such as when application containers are unable to use persistent volumes, you can configure the transaction management to store transaction logs in a database by using a JDBC datasource.
 
-IMPORTANT: While there are several benefits to using a database to store transaction logs, you might notice a reduction in performance compared with using the file system to store the logs.
+NOTE: This configuration is only relevant for _Jakarta Transactions_ transactions.
 
-Quarkus allows the following JDBC-specific configuration of the object store included in `quarkus.transaction-manager.object-store.<property>` properties, where <property> can be:
+However, in cloud-native apps, it is important to assess JTA transactions after careful evaluation.
```

Comment:
The plan was to define what transaction model the extension supports in the introductory paragraph and thereafter to just use the term transaction. I think Manuel also highlighted the use of `JTA` in one of his earlier comments.